src/js/ripple/bignumber.js

```diff
@@ -0,0 +1,28 @@
+var BigNumber = require('bignumber.js');
+var extend = require('extend');
+
+function BigNumberWrapper(value, base) {
+  // reset config every time a BigNumber is instantiated so that
```

Comment:
If we're going to go to some troubles, maybe we should go the full monty. This reconfigures before every construction, but not operations. Is this going to be enough ? ( haven't looked myself )